extensions/liquibase/runtime/src/main/java/io/quarkus/liquibase/runtime/LiquibaseDataSourceBuildTimeConfig.java

```diff
@@ -28,4 +32,9 @@ public static final LiquibaseDataSourceBuildTimeConfig defaultConfig() {
     @ConfigItem(defaultValue = DEFAULT_CHANGE_LOG)
     public String changeLog;
 
+    /**
+     * The search path for DirectoryResourceAccessor
+     */
+    @ConfigItem(defaultValue = "/")
```

Comment:
Yeah I saw that but I have to admit it's confusing. Anyway, let's leave it like this for now, but I assume that using `Optional<List<String>>` for `searchPath` would be better.

Reply:
I did it this way because at the time I could not find how to declare the default list as an empty list in the build time properties, because if I put a default value of "" in the tag, that is an empty string, I got an error related to null.

Reply:
```java
    /**
     * The search path for DirectoryResourceAccessor
     */
    @ConfigItem
    public Optional<List<String>> searchPath;
```

should work, no?

Reply:
Let's merge this and you can try and update both configs with my suggestion. If all goes well, you can open a new PR